README.md

```diff
@@ -23,7 +23,7 @@ Project source code: [https://github.com/jarviscanada/jarvis_data_eng_JasmeetBra
   - JDBC App: Created an application that would access a database and perform queries using JDBC. Java and JDBC APIs were used to create the DAO necessary to perform operations with a Postgres database running on Docker, and to segregate these procedures from business side logic.
   - Grep App: Constructed a replica of the grep command line utility. Used Java along with Lambda and Stream APIs as part of the implementation, and used Maven to package the application, which was later deployed on Docker Hub using Docker.
 
-**Springboot App** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_JasmeetBrar/tree/master/springboot)]: Not Started
```

Comment:
please run a grammarly check for the entire profile
![image](https://user-images.githubusercontent.com/58951265/120226815-ed85c200-c215-11eb-9e97-2d082653f523.png)

Reply:
Hi, I'm not getting the highlights that you are getting there. What settings did you have set up for Grammarly?

Reply:
I reviewed my profile, and so far everything does look good for now. I'll merge, and make any needed adjustments after today.